### ISSUE TITLE
训练数据labels打印出来全是-100

### ISSUE THREAD

**Issue by fengzi00** [THUDM/ChatGLM-6B#686]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

执行 bash train.sh
打印labels全是-100，训练完成之后调用模型，输出的全是空

### Steps To Reproduce

1.训练数据 参考的示例
2.模型训练成功，训练完成也能调用。查看日志发现打印的labels全是-100，调用pt模型，输出为空

### Environment

```markdown
- OS:centos7
- Python:3.9
- Transformers:4.27.1
- PyTorch:1.13
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :11.8
```

**Comment by duzx16** [comment]
-100 是表示这个位置的 token 不需要计算 loss。你确定全部都是 -100 吗？可以发一下截图

**Comment by fengzi00** [comment]
![image](https://user-images.githubusercontent.com/50097334/232946297-fddfad78-4d63-4950-8b74-a108cc383b5e.png)
![image](https://user-images.githubusercontent.com/50097334/232946336-cdc143b4-d359-4c79-8a0b-2ea71f7ef3f1.png)

**Comment by fengzi00** [comment]
> -100 是表示这个位置的 token 不需要计算 loss。你确定全部都是 -100 吗？可以发一下截图

看着像是代码有问题，输入输出全部放入inputs了

**Comment by yc930401** [comment]
遇到同样问题！

**Comment by duzx16** [comment]
> ![image](https://user-images.githubusercontent.com/50097334/232946297-fddfad78-4d63-4950-8b74-a108cc383b5e.png) ![image](https://user-images.githubusercontent.com/50097334/232946336-cdc143b4-d359-4c79-8a0b-2ea71f7ef3f1.png)

模型实现太旧了。如果你是从 `THUDM/chatglm-6b` 加载的话可以清除一下 `~/.cache/huggingface/modules/transformers_modules`
